### PR TITLE
Add support for passive events

### PR DESCRIPTION
**Pull request by Ceane**
With Chrome launching [passive events](https://dom.spec.whatwg.org/#dom-eventlisteneroptions-passive) in [Chrome 51](https://www.chromestatus.com/features/5745543795965952), I think it would be appropriate to support this functionality in RxJS.

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@Ceane__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no Contribution License Agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, MSOTBOT;

File: src/core/linq/observable/fromevent.js [hunk 1]
```diff
@@ -77,15 +77,16 @@
    * Creates an observable sequence by adding an event listener to the matching DOMElement or each item in the NodeList.
    * @param {Object} element The DOMElement or NodeList to attach a listener.
    * @param {String} eventName The event name to attach the observable sequence.
+   * @param {Object} eventListenerOptions An object describing EventListenerOptions
    * @param {Function} [selector] A selector which takes the arguments from the event handler to produce a single item to yield on next.
    * @returns {Observable} An observable sequence of events from the specified element and the specified event.
    */
-  Observable.fromEvent = function (element, eventName, selector) {
+  Observable.fromEvent = function (element, eventName, eventListenerOptions, selector) {
```
Comment (paulpdaniels) [inline src/core/linq/observable/fromevent.js]: This is a breaking change. The `eventListenerOptions` either needs to go after the `selector` or there needs to be logic to detect the original scenario of passing the `selector` function as the 3rd argument.

File: src/core/linq/observable/fromevent.js [hunk 2]
```diff
@@ -77,15 +77,16 @@
...
     // Node.js specific
     if (element.addListener) {
       return fromEventPattern(
-        function (h) { element.addListener(eventName, h); },
-        function (h) { element.removeListener(eventName, h); },
+        function (h) { element.addListener(eventName, h, eventListenerOptions); },
+        function (h) { element.removeListener(eventName, h, eventListenerOptions); },
```
Comment (paulpdaniels) [inline src/core/linq/observable/fromevent.js]: I don't understand, you said this is being added to the Chrome 51 spec, but the only places you have changed are non-native adapter code for Node and several popular frameworks.

File: src/core/linq/observable/fromevent.js (hunk 1 above)
Comment (paulpdaniels) [inline src/core/linq/observable/fromevent.js]: You should probably add unit tests and update the docs as well.

Comment (Ceane) [inline src/core/linq/observable/fromevent.js]: Sure thing

File: src/core/linq/observable/fromevent.js (hunk 2 above)
Comment (Ceane) [inline src/core/linq/observable/fromevent.js]: My apologies, I will get around to this later today

**Comment by msotclas** [comment]
__@Ceane__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, MSOTBOT;

**Pull request merged**